website/README.md

```diff
@@ -5,21 +5,26 @@ This website is built using [Docusaurus 2](https://docusaurus.io/), a modern sta
 ## Installation
 
 ```console
-yarn install
+npm install
 ```
 
 ## Local Development
 
 ```console
-yarn start
+npm start
 ```
 
-This command starts a local development server and opens up a browser window. Most changes are reflected live without having to restart the server.
+This command starts a local development server and opens up a browser window. Most changes are reflected live
+without having to restart the server.
+Note this only builds for English locale unlike a production build.
 
-## Build
+> Documentation is written in `mdx`, markdown empowered with jsx.
+> JetBrains and VSCode both provide MDX plugins.
+
```

Comment:
This is worth noting, future doc contributors will benefit hugely from MDX intellisense help.